File: src/tests/BusinessObjects/AuthorPersistentObject.cs
using System;
using System.Data;
using voidsoft.DataBlock;
namespace Extender
{
public class AuthorPersistentObject : voidsoft.DataBlock.PersistentObject
{ 
 
      public AuthorPersistentObject(DatabaseServer database, string connectionString, TableMetadata mainTable) : base (database, connectionString, mainTable) 
      {
      }
      
      public AuthorPersistentObject(Session session, TableMetadata mainTable) : base(session, mainTable)
      {
      }

    public AuthorPersistentObject(TableMetadata m)
        : base(m)
    {

    }
      
}
}

File: src/tests/BusinessObjectsVB/AuthorPersistentObject.vb
Imports System
Imports System.Data
Imports voidsoft.DataBlock
Namespace Extender
 
Public Class AuthorPersistentObject
								Inherits PersistentObject 
 
      Public Sub New (database as DatabaseServer,connectionString as String,mainTable as TableMetadata)
               MyBase.New(database, connectionString, mainTable)
      End Sub
	
      Public Sub New (s as Session, mainTable as TableMetadata)
               MyBase.New(s, mainTable)
      End Sub
      Public Sub New (mainTable as TableMetadata)
               MyBase.New(mainTable)
      End Sub
	
End Class
End Namespace
